src/test/jtx/Env_test.cpp

```diff
@@ -702,7 +703,7 @@ class Env_test : public beast::unit_test::suite
             // the two supported ones
             Env env{
                 *this,
-                with_only_features(
+                FeatureBitset(
```

Comment:
in-keeping with the other changes, you could just remove `neverSupported` and use `*neverSupportedFeat` directly in a call to `.set()`

Reply:
`set` is not a `const` function, so that would change the collection. However, I did refactor this a bit.